sklearn/linear_model/tests/test_ridge.py

```diff
@@ -1234,18 +1234,31 @@ def test_ridgecv_int_alphas():
     ridge.fit(X, y)
 
 
-def test_ridgecv_negative_alphas():
+def test_ridgecv_scalar_alphas():
```

Comment:
This test is very well written. :+1: 

Could it be parametrized or extended to test the correct validation on `RidgeClassifierCV`?

Reply:
Thanks! Done!